packages/expo/src/environment/DevLoadingView.tsx

```diff
@@ -101,11 +101,9 @@ export default function DevLoadingView() {
 /**
  * This is a hack to get the safe area insets without explicitly depending on react-native-safe-area-context.
  * The following code is lifted from: https://git.io/Jzk4k
- *
- * TODO: This will need to be updated for Fabric/TurboModules.
  **/
-const RNCSafeAreaProviderConfig = UIManager.getViewManagerConfig('RNCSafeAreaProvider') as any;
-const initialWindowMetrics = RNCSafeAreaProviderConfig?.Constants?.initialWindowMetrics as
+const RNCSafeAreaContext = NativeModules.RNCSafeAreaContext;
```

Comment:
Intentional not to use turbomodules?
```tsx
const RNCSafeAreaContext = TurboModuleRegistry.get('RNCSafeAreaContext');
RNCSafeAreaContext?.getConstants().initialWindowMetrics;
```

Reply:
no good reason outside of old habits

Reply:
Probably better to use the new interface then, which I assume is more likely to last?

Reply:
`TurboModuleRegistry.get` does support legacy bridge modules when remote debugger is on, so yeah, that sounds like a better solution